File: codequest/2019Solutions/Lockheed Solutions/Python/Prob06.py
# Recommended imports for all problems
# Some problems may require more
import sys
import math
import string

# Always start with reading in the number
# of test cases from standard input. The
# rstrip() method removes the lingering newline
cases = int(sys.stdin.readline().rstrip())
# Loop for each test case. This is the last line
# common to all problems; the content of this
# loop will change based on the problem solved.
for caseNum in range(cases):
        altitude = int(sys.stdin.readline().rstrip());
        circumference = (2 * math.pi * altitude) + 40075;
        circumference = round(circumference * 10.0) / 10.0;
        print(circumference);

File: codequest/2019Solutions/Lockheed Solutions/C/Prob06.c
#include <stdio.h>
#include <math.h>

int main() {

    double earth_circumference;
    earth_circumference = 40075;

    int num_inputs;

    scanf("%d", &num_inputs);
    
    double earth_radius;
    earth_radius = earth_circumference/2.0/M_PI;

    double radius;
    double circumference;
    double altitude;
    
    int i;
    for (i = 0; i < num_inputs; i++) {

        scanf("%lf", &altitude);
        radius = earth_radius + altitude;

        circumference = 2.0*M_PI*radius;
        printf("%.1f\n", round(circumference*10.0)/10.0);
    }
}
